### ISSUE TITLE
FR Radio: Add search results to playlist

### ISSUE THREAD

**Issue by gerroon** [fatg3erman/RompR#44]
HI

Thanks for integrating radio-browser. It seems to be natural fit for RompR.  

It would be very nice if there is a way to add all search results to the current playlist at once.

**Comment by tljohnsn** [comment]
Agreed.  I was looking for a way do do this as well.  The best workaround I found was to select "Display results as->Directory Tree" and then you can add the item from the tree.  This does not work well however when searching by stars.  For some reason with stars and directory tree, only one result is ever displayed.
